src/NLog.Targets.Syslog/AsyncLogger.cs

```diff
@@ -116,20 +116,15 @@ private Task ProcessQueueAsync(MessageBuilder messageBuilder, TaskCompletionSour
 
         private void Enqueue(AsyncLogEventInfo asyncLogEventInfo, int timeout)
         {
-            void LogEnqueueResult(string message)
-            {
-                if (!InternalLogger.IsDebugEnabled)
-                    return;
-                InternalLogger.Debug("[Syslog] {0} '{1}'", message, asyncLogEventInfo.ToFormattedMessage());
-            }
+            bool success = queue.TryAdd(asyncLogEventInfo, timeout, token);
 
-            if (queue.TryAdd(asyncLogEventInfo, timeout, token))
+            if (InternalLogger.IsDebugEnabled)
             {
-                LogEnqueueResult("Enqueued");
-                return;
+                InternalLogger.Debug("[Syslog] {0} '{1}'", success ? "Enqueued" : "Failed Enqueue", asyncLogEventInfo.ToFormattedMessage());
```

Comment:
Nitpick: `Failed Enqueue` to `Failed enqueuing`

Reply:
Changed